pyproject.toml

```diff
@@ -26,10 +26,10 @@ include = [
 [tool.poetry.dependencies]
 python = "^3.7"
 cython = "^0.29.14"
-google-cloud-storage = ">=1.24.1 <=1.28.1"
+google-cloud-storage = ">=1.31.1"
```

Comment:
I am assuming this [issue](https://github.com/Unity-Technologies/datasetinsights/pull/82) was fixed in `google-cloud-storage 1.31+?

Reply:
i did not know about that issue. it looks like 1.31.1 merged those changes. we could do a later version if that matters, but i picked this to align with what sprawl uses currently

release notes for the fix: https://github.com/googleapis/python-storage/releases/tag/v1.31.1
`blob: base64 includes additional characters`

Reply:
Good to know the issue is fixed. Let's also include `<2.0.0` in the requirements to make sure we don't accidentally bump the major version.

Reply:
updated. anything look off?